classes/task/emit_task.php

```diff
@@ -50,10 +50,13 @@ public function execute() {
         $store = new store($manager);
 
         $events = $DB->get_records('logstore_xapi_log');
-        $store->process_events($events);
-
-        $DB->delete_records_list('logstore_xapi_log', 'id', array_keys($events));
-
+        $store_return = $store->process_events($events);
+		foreach(array_keys($store_return) as $event_id)
+			{
```

Comment:
Opening brace needs to be on the same line as the `foreach` according to the [style guide](http://jlowe64.github.io/moodle-logstore_xapi/).